shopify-customer-to-salesforce-customer/main.bal

```diff
@@ -0,0 +1,46 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/regex;
+import ballerinax/salesforce as sf;
+
+configurable sf:ConnectionConfig salesforceConfig = ?;
+sf:Client salesforce = check new (salesforceConfig);
+
+service /salesforce_bridge on new http:Listener(9090) {
```

Comment:
Shall we use hiphen instead of underscore in `salesforce_bridge` as that increases the readability of URL.  Refer .https://learn-ballerina.github.io/best_practices/naming_conventions.html#rest-service-and-resource-paths

Reply:
IIRC, having a hiphen in service names gives an error.